ctx_interface.go

```diff
@@ -232,13 +232,6 @@ type Ctx interface {
 	// Protocol returns the HTTP protocol of request: HTTP/1.1 and HTTP/2.
 	Protocol() string
 
-	// Query returns the query string parameter in the url.
-	// Defaults to empty string "" if the query doesn't exist.
-	// If a default value is given, it will return that value if the query doesn't exist.
-	// Returned value is only valid within the handler. Do not store any references.
-	// Make copies or use the Immutable setting to use the value outside the Handler.
-	Query(key string, defaultValue ...string) string
```

Comment:
we still need Query in the interface, but with a different signature

Reply:
about this do I add it again back in the interface?

Reply:
yes pls, but with adjusted documentation